src/jobs/time-tracking-job.js

```diff
@@ -11,30 +11,48 @@
  *
  */
 
+const moment = require('moment');
+
 const BaseJobHandler = require('./base/base-job-handler');
+const FogAccessTokenService = require('../services/iofog-access-token-service');
+const logger = require('../logger');
 const Tracking = require('../tracking');
 const TrackingEventType = require('../enums/tracking-event-type');
+const TransactionDecorator = require('../decorators/transaction-decorator');
+
+
+const INTERVAL = 5 * 60 * 1000;
 
 class TimeTrackingJob extends BaseJobHandler {
 
   constructor() {
     super();
-    this.scheduleTime = intervalMin * 60 * 1000;
+    this.startTime = moment.now();
   }
 
   run() {
-    setInterval(trackTime, this.scheduleTime);
+    setTimeout(this.trackTime, INTERVAL);
   }
-}
 
-let iteration = 0;
-const intervalMin = 5;
+  async trackTime() {
+    let agentsCount = 0
+    try {
+      const agents = await TransactionDecorator.generateFakeTransaction(FogAccessTokenService.all)();
+      agentsCount = (agents || []).length;
+    } catch (e) {
+      logger.warn('Unable to count ioFog agents')
+    }
 
-async function trackTime() {
-  iteration++;
-  const runningTime = iteration * intervalMin;
-  const event = Tracking.buildEvent(TrackingEventType.RUNNING_TIME, runningTime,);
-  await Tracking.processEvent(event);
+    try {
+      const runningTime = moment().diff(this.startTime, 'minutes');
+      const event = Tracking.buildEvent(TrackingEventType.RUNNING_TIME, { runningTime, agentsCount });
+      await Tracking.processEvent(event);
+    } catch (err) {
+      logger.error(`Unable to send "${TrackingEventType.RUNNING_TIME}" tracking info`);
```

Comment:
Tracking.processEvent() for timeTracking stores data in local DB. This data will be sent later by send-tracking-job.js, so this catch looks unnecessary or message should be changed